courses.md

```diff
@@ -1,5 +1,10 @@
 ## Courses, Workshops, and Tutorials
+
+* [SciPy 2018](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=26&cad=rja&uact=8&ved=0ahUKEwjKreXT_abcAhULxVQKHbPsAKkQtwIIYjAZ&url=https%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DkJXUUO5M4ok&usg=AOvVaw0fMlkj1OkQEtJZJpOw2t4P)
```

Comment:
I think this might not be right as a google.com url?

Reply:
Try this https://www.youtube.com/watch?v=kJXUUO5M4ok